deployments/operator/webhook/manifests.yaml

```diff
@@ -133,7 +153,6 @@ webhooks:
       name: webhook-service
       namespace: system
       path: /pods-sgx
-  reinvocationPolicy: IfNeeded
```

Comment:
`controller-gen` is not working as we want and it removes this line. We want to keep it so please drop this removal.

Reply:
This is really weird..
In my visual code, it does not show this removed part..
Even in the commit I made earlier the removed line exsited,, but if I push,, then it suddenly changes..

I just reset the branch with mixed option and re-wrote copying by hand.